code/modules/vehicles/motorized_wheelchair.dm

```diff
@@ -0,0 +1,153 @@
+/obj/vehicle/ridden/wheelchair/motorized
+	name = "motorized wheelchair"
+	desc = "A chair with big wheels. It seems to have a motor in it."
+	max_integrity = 150
+	var/speed = 2
+	var/power_efficiency = 1
+	var/power_usage = 100
+	var/panel_open = FALSE
+	var/list/required_parts = list(/obj/item/stock_parts/manipulator, 
+							/obj/item/stock_parts/manipulator,
+							/obj/item/stock_parts/capacitor)
+	var/obj/item/stock_parts/cell/power_cell
+
+/obj/vehicle/ridden/wheelchair/motorized/CheckParts(list/parts_list)
+	..()
+	refresh_parts()
+
+/obj/vehicle/ridden/wheelchair/motorized/proc/refresh_parts()
+	var/_temp = 0
+	for(var/obj/item/stock_parts/manipulator/M in contents)
+		_temp += M.rating
+	speed = _temp
```

Comment:
You don't need a temp var when speed and power_efficiency get reset anyways. just zero both on refresh parts.

also since you've done it this way, the first manip's effects get canceled out by the second one.

Reply:
This is not as efficient in the cap calculation since you're choosing

rating > temp > power

instead of

rating > power

Reply:
there